sechub-shared-kernel/src/test/java/com/mercedesbenz/sechub/sharedkernel/messaging/ListOfStringsMessageDataProviderTest.java

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.sharedkernel.messaging;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+class ListOfStringsMessageDataProviderTest {
+
+    private ListOfStringsMessageDataProvider providerToTest = new ListOfStringsMessageDataProvider();
+
```

Comment:
The tests are bit strange to me:

/* execute */ block does contain always 2 calls - which one is the one you test? For me it looks like these are always two tests combined.

And I would have written the tests with one or two ArguementsProvider and made the tests parameterized: provider would just have input and expected output - and maybe an identifier as first parameter, so better to read in outputs.

LocalDateTimeMessageDataProviderTest